### ISSUE TITLE
npm test does not work

### ISSUE THREAD

**Issue by frankgerhardt** [jupyter-widgets/widget-ts-cookiecutter#53]
```
$ npm test

> fastawidget@0.1.0 test /home/s/git/p/ipywidgets/fastawidget
> npm run test:firefox


> fastawidget@0.1.0 test:firefox /home/s/git/p/ipywidgets/fastawidget
> karma start --browsers=Firefox tests/karma.conf.js


START:
12 10 2019 16:51:22.236:INFO [compiler.karma-typescript]: Compiling project using Typescript 3.1.6
12 10 2019 16:51:23.725:ERROR [compiler.karma-typescript]: tests/src/utils.spec.ts(24,46): error TS2694: Namespace '"/home/s/git/p/ipywidgets/fastawidget/node_modules/@jupyterlab/services/lib/kernel/messages".KernelMessage' has no exported member 'ICommMsg'.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! fastawidget@0.1.0 test:firefox: `karma start --browsers=Firefox tests/karma.conf.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fastawidget@0.1.0 test:firefox script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/siena/.npm/_logs/2019-10-12T14_51_23_740Z-debug.log
npm ERR! Test failed.  See above for more details.
```
I have no idea. It does not look like "This should again, only give TODO errors (Expected 'Value' to equal 'Expected value')"

**Comment by deeplook** [comment]
Running into the same thing... The tests won't execute, but the code will install. ;)

**Comment by vidartf** [comment]
I fixed this recently somewhere else. I think these are the needed steps (might be more though):
- Add this dev dep to JS package: `"karma-typescript-es6-transform": "^4.1.1",`
- Add these options to the karma config:
```js
      bundlerOptions: {
        acornOptions: {
          ecmaVersion: 8,
        },
        transforms: [
          require("karma-typescript-es6-transform")({
            presets: [
              ["env", {
                targets: {
                  browsers: ["last 2 Chrome versions"]
                },
              }]
            ]
          })
        ]
      }
```

**Comment by vidartf** [comment]
If someone could test if those are sufficient, and then do a PR if that's the case, that would be very helpful!

**Comment by ianhi** [comment]
I think this was fixed in #69
